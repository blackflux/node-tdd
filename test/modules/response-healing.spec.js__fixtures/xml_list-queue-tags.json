
{
  "fn": "call",
  "params": [
    "sqs:ListQueueTagsCommand",
    {"QueueUrl": "$queueUrl"}
  ],
  "cassette": [
    {
      "scope": "https://sqs.us-west-2.amazonaws.com:443",
      "method": "POST",
      "path": "/",
      "body": "QueueUrl=https%3A%2F%2Fsqs.us-west-2.amazonaws.com%2F012345678910%2Fsome-queue-url&Action=ListQueueTags&Version=2012-11-05",
      "status": 200,
      "response": "<?xml version=\"1.0\"?><ListQueueTagsResponse xmlns=\"http://queue.amazonaws.com/doc/2012-11-05/\"><ListQueueTagsResult><Tag><Key>tagA</Key><Value>4cbc382c-94d0-48fb-b625-f098683afd3d</Value></Tag><Tag><Key>tagB</Key><Value>ccfc9f9b-5379-4ae7-ba5c-5891a8259f83</Value></Tag><Tag><Key>lastModified</Key><Value>2022-02-10T21:54:53.039Z</Value></Tag></ListQueueTagsResult><ResponseMetadata><RequestId>b3fee173-056e-5034-bd15-bbf9244fa0de</RequestId></ResponseMetadata></ListQueueTagsResponse>",
      "responseIsBinary": false
    }
  ]
}
