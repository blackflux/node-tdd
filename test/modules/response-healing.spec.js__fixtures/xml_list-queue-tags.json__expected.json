[
  {
    "scope": "https://sqs.us-west-2.amazonaws.com:443",
    "method": "POST",
    "path": "/",
    "body": {
      "QueueUrl": "https://sqs.us-west-2.amazonaws.com/123456789101/service-name-data-local-SomeQueue"
    },
    "status": 200,
    "response": {
      "Tags": {
        "lastModified": "2022-02-10T21:54:53.039Z",
        "tagB": "ccfc9f9b-5379-4ae7-ba5c-5891a8259f83",
        "tagA": "4cbc382c-94d0-48fb-b625-f098683afd3d"
      }
    },
    "responseIsBinary": false,
    "reqheaders": {
      "amz-sdk-invocation-id": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$",
      "amz-sdk-request": "attempt=1; max=3",
      "authorization": "^AWS4-HMAC-SHA256 Credential=.+$",
      "content-length": "^[1-9][0-9]*$",
      "content-type": "application/x-amz-json-1.0",
      "host": "sqs.us-west-2.amazonaws.com",
      "user-agent": "^aws-sdk-js/.+$",
      "x-amz-content-sha256": "^[a-f0-9]{64}$",
      "x-amz-date": "^[0-9]{8}T[0-9]{6}Z$",
      "x-amz-target": "AmazonSQS.ListQueueTags",
      "x-amz-user-agent": "^aws-sdk-js/[0-9]+.[0-9]+.[0-9]+$",
      "x-amzn-query-mode": "true",
      "connection": "close"
    }
  }
]
